docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -62,6 +65,9 @@ who created or updated them.
 
 --
 
+`read_ccr`::
+TBD
```

Comment:
Roughly: This is a cluster privilege needed on a cluster holding leader indices to grant permissions to the user to read what indices are in the cluster, read leader index metadata, and check that the user has the appropriate privileges to follow leader indices (@martijnvg double check me here please).

Reply:
Thanks, @jasontedor , I've drafted descriptions for these.